app/views/discovered_mailer/discovered_summary.html.erb

```diff
@@ -4,23 +4,68 @@
 <head>
   <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
   <title><%= _('Summary report for discovered hosts from Foreman') %></title>
+  <style>
```

Comment:
core uses a different approach using the roadie-rails gem. if you don't want to do it as part of this pr, please revert these changes?

Reply:
missed that, sorry, I'm fixing it

Reply:
Add the change, did not test it yet, I will notify when it is ready